src/database/entities/applications.entity.ts

```diff
@@ -42,4 +42,6 @@ export class ApplicationEntity {
 
   @UpdateDateColumn({ update: true })
   updated_at: Date;
+  status: any;
```

Comment:
O ideal seria um enum.

Reply:
Eu tentei criar mas não sei se as opções ideiais seriam as que eu coloquei, se não for me avisa que aí agora criado depois é só eu alterar ali, tranquilo.

Reply:
Verifica por favor:
- As redundâncias dos IDs, existe campos que está ocupando espaço desnecessário no BD
- Não é o ideial deixar o status no "any"
- Verifica na linha 58 está com erro de digitação.
- Verifica também na documentação algumas validações, como por exemplo, validação de email, formato de email, ou tamanho max do campo de texto.